src/client/components/app/DetailCard/index.tsx

```diff
@@ -45,22 +57,25 @@ const StyledCardContent = styled(CardContent)<{ wrap?: boolean; pointer?: boolea
   &:hover {
     background-color: ${({ theme }) => theme.colors.selectionBar};
 
-    ${StyledCardElement} {
-      color: ${({ theme }) => theme.colors.onSurfaceH2Hover};
-    }
-    .action-button {
-      background: ${({ theme }) => theme.colors.vaultActionButton.selected.background};
-      color: ${({ theme }) => theme.colors.vaultActionButton.selected.color};
-      border: 2px solid ${({ theme }) => theme.colors.vaultActionButton.selected.borderColor};
-    }
+    // NOTE If you want to change other elements on selection bar hover
+    // ${StyledCardElement} {
+    //   color: ${({ theme }) => theme.colors.titles};
+    // }
+    // .action-button {
+    //   background: ${({ theme }) => theme.colors.vaultActionButton.selected.background};
+    //   color: ${({ theme }) => theme.colors.vaultActionButton.selected.color};
+    //   border: 2px solid ${({ theme }) => theme.colors.vaultActionButton.selected.borderColor};
+    // }
   }
 `;
 
 const StyledCardHeader = styled(CardHeader)`
   display: flex;
-  flex-wrap: center;
+  flex-wrap: wrap;
+  align-items: center;
   justify-content: space-between;
-  margin-bottom: 0.6rem;
+  margin-bottom: 2rem;
+  grid-gap: 1.2rem;
```

Comment:
Grid-gap has backward compatibility when gap doesn't, also works with flexbox specially in safari, and the polyfill doesn't create a new one. Normally we should switch every gap to grid-gap

Reply:
It's widely supported as `gap` since 2018 according to can-i-use

Reply:
Check for grid layout and for flexbox. 
We had this problem with scroll on safari on banteg's old osx. 
I can show you an example in a call. but gap works after safari 14, and grid-gap works since 2018. Gap does not when I tested it

Reply:
<img width="853" alt="image" src="https://user-images.githubusercontent.com/78362532/162554121-b3d2f021-4872-4311-adb3-03e369f99b01.png">